client/my-sites/post-type-list/post-actions-ellipsis-menu/view.jsx

```diff
@@ -42,6 +42,7 @@ class PostActionsEllipsisMenuView extends Component {
 
 		this.props.setPreviewUrl( previewUrl );
 		this.props.setLayoutFocus( 'preview' );
+		this.props.onClick();
```

Comment:
>Because `PostActionsEllipsisMenuView` never does anything with the onClick prop it is being passed.

Okay, took another look and this makes a little more sense; the `onClick` prop is passed to the `PopoverMenuItem` but I'm guessing it gets lost in the fact that the child of `PopoverMenu` is not the menu item directly, but rather `PostActionsEllipsisMenuView`.

>Should I specify default props in `PostActionsEllipsisMenuView` then?

Yeah, that seems fine. Seems like it'll always be assigned anyways, but best to be safe.